src/segment_regquery_windows_test.go

```diff
@@ -0,0 +1,94 @@
+//go:build windows
+
+package main
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestRegQueryGetRegistryFunction(t *testing.T) {
```

Comment:
this is an integration test, not a functionality test for the segmenmt. It's good, but needs to move the tests oof environment (on windows)

Reply:
test code still needs sorting out - currently commented out